eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -60,7 +60,7 @@ stages:
                     inputs:
                       solution: '${{ parameters.BuildToolsPath }}/tools/symboltool/SymbolUploader.proj'
                       msbuildArguments: >
-                        /p:PackagesPath='$(PIPELINE.WORKSPACE)\packages'
+                        /p:PackagesPath=$(PIPELINE.WORKSPACE)\packages
```

Comment:
Do you plan to revert the slashes as well now? Also is it just the single quotes issue, should these be double quotes?

Reply:
I can revert the slashes. I think double quotes will be fine but am more confident with passing no quotes as that how we have done it else where.

Reply:
Passing no quotes works fine as long as the paths have not spaces in them.